integrations/ashby/nango.yaml

```diff
@@ -38,6 +38,72 @@ integrations:
                 endpoint:
                     method: POST
                     path: /notes
+            application-change-source:
+                output: AshbyResponse
+                input: ChangeSource
+                description: |
+                    Action to change source of application.
+                endpoint:
+                    method: POST
+                    path: /applications/source
+                    group: Applications
+                scopes:
+                    - candidatesWrite
+            application-change-stage:
+                output: AshbyResponse
+                input: ChangeStage
+                description: |
+                    Action to change stage of an application.
+                endpoint:
+                    method: POST
+                    path: /applications/stage
+                    group: Applications
+                scopes:
+                    - candidatesWrite
+            application-update:
+                output: AshbyResponse
+                input: ChangeSource | ChangeStage | UpdateHistory
+                description: |
+                    Action to update an application stage.
+                endpoint:
+                    method: PATCH
+                    path: /applications/update
+                    group: Applications
+                scopes:
+                    - candidatesWrite
+            application-update-history:
+                output: AshbyResponse
+                input: UpdateHistory
+                description: |
+                    Action to update history an application stage.
+                endpoint:
+                    method: POST
+                    path: /applications/history
+                    group: Applications
+                scopes:
+                    - candidatesWrite
+            create-candidate:
+                output: AshbyResponse
+                input: CreateCandidate
+                description: |
+                    Action to update history an application stage.
+                endpoint:
+                    method: POST
+                    path: /candidates/create
```

Comment:
```
path: /candidates
group: Candidates
```

Reply:
@dannylwe looks like this change was missed. Can you make an issue and follow up on this item